### PR TITLE
[UI] Environments page and related components MUI v5 migration

### PR DESCRIPTION
**Pull request by SAHU-01**
**Notes for Reviewers**

- This PR fixes migrates the Environments page code to use MUI v5 styling and sistent theme and components

https://github.com/user-attachments/assets/206760b8-e50a-424b-a018-048824a33066

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Lifecycle/Environments/index.js [hunk 1]
```diff
@@ -78,9 +80,11 @@ const Environments = ({ organization, classes }) => {
   const pageSize = 10;
   const connectionPageSize = 25;
 
+  const theme = useTheme();
+
   const modalRef = useRef(null);
   const { notify } = useNotification();
-  const StyleClass = useStyles();
+  // const StyleClass = useStyles();
```
Comment (sudhanshutech) [inline ui/components/Lifecycle/Environments/index.js:87]: remove if not needed

File: ui/components/Lifecycle/Environments/styles.js [hunk 2]
```diff
@@ -1,285 +1,265 @@
...
-    borderRadius: '5px 0 0 5px',
-    padding: '8px 16px',
-    borderRight: '0',
-    '&:hover': {
-      cursor: 'default',
-    },
+export const TabIconBox = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  border: `1px solid #00B39F`,
```
Comment (sudhanshutech) [inline ui/components/Lifecycle/Environments/styles.js:60]: border token should be used here

File: ui/components/Lifecycle/Environments/styles.js [hunk 3]
```diff
@@ -1,285 +1,265 @@
...
-    padding: '8px 16px',
-  },
+export const TabNameBox = styled('div')(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  flexDirection: 'column',
+  background: '#00B39F',
+  color: '#fff',
+  padding: '8px 16px',
+  border: `1px solid #00B39F`,
```
Comment (sudhanshutech) [inline ui/components/Lifecycle/Environments/styles.js:76]: same here

File: ui/components/Lifecycle/Environments/styles.js [hunk 4]
```diff
@@ -1,285 +1,265 @@
...
+});
 
-  iconButton: {
-    minWidth: 'fit-content',
-    '&:hover': {
-      background: 'transparent',
-    },
-    padding: 0,
+export const EditButton = styled('button')({
+  backgroundColor: '#00B39F',
```
Comment (sudhanshutech) [inline ui/components/Lifecycle/Environments/styles.js:240]: use the standard button and background token

**Comment by sudhanshutech** [comment]
why the eslint check failing @SAHU-01? .... and did something wrong with the placement of search.

**Comment by sudhanshutech** [comment]
please take care of above feedbacks , otherwise looks good.

**Comment by SAHU-01** [comment]
> why the eslint check failing @SAHU-01? .... and did something wrong with the placement of search.

Not sure about the eslint checks @sudhanshutech

**Pull request closed without merging**